conda-store-server/tests/test_actions.py

```diff
@@ -205,3 +212,83 @@ def test_add_lockfile_packages(
 
     solve = api.get_solve(db, solve_id=solve_id)
     assert len(solve.package_builds) > 0
+
+
+@pytest.mark.parametrize(
+    "key",
+    [
+        "",  # legacy build
+        "non-empty",  # new build
```

Comment:
Works with me. There is a property on builds which auto computes this key https://github.com/conda-incubator/conda-store/blob/1b88b509f52473f93371135b118717949ee6e188/conda-store-server/conda_store_server/orm.py#L246. But I don't think this matters.

Reply:
I disagree, and I think it _does_ matter regardless of being part of a test so would suggest giving this a more meaningful name

Reply:
It's not possible to hardcode the key value for new-style builds because part of it changes every run (date). I refactored the test to get this value from `conda_lock_key`. Also added a check to make sure it matches the returned location header.